ExampleMarkdownDocument.md

```diff
@@ -0,0 +1,93 @@
+# X-Road: Example Markdown Document
+
+Version: 1.0.2  
+Doc. ID: UG-EXAMPLEMD  
+
+| Date       | Version     | Description                                     | Author             |
+|------------|-------------|-------------------------------------------------|--------------------|
+| 13.01.2017 | 1.0.0       | Initial version                                 | Vitali Stupin      |
+| 13.01.2017 | 1.0.1       | Fixed some typos                                | Vitali Stupin      |
+| 20.01.2017 | 1.0.2       | Fixed some more typos                           | Vitali Stupin      |
+
+## Table of Contents
+
+[Table of Contents](#table-of-contents)  
+[License](#license)  
+[1 Introduction](#1-introduction)  
+&nbsp;&nbsp;&nbsp;&nbsp;[1.1 Terms and Abbreviations](#11-terms-and-abbreviations)  
+&nbsp;&nbsp;&nbsp;&nbsp;[1.2 References](#12-references)  
+[2 Examples](#2-examples)  
+[Annex A Example XML](#annex-a-example-xml)  
+
+## License
+
+This work is licensed under the Creative Commons Attribution-ShareAlike 3.0 Unported License.
```

Comment:
This is a nice question. For me this change is acceptable. But we would need to update all documents then. because currently we use "This work" in all X-Road documents.